bin/yaml/glsl.yaml

```diff
@@ -0,0 +1,10 @@
+compilers:
+  glsl:
+    glslang:
+      type: ziparchive
+      dir: glslang-{name}
```

Comment:
As it stands this will install once (we don't reinstall all compilers every day, for obvious reasons :))

If you want it to install every night you'll need to add:

```
if: nightly
install_always: true
```

here.

Reply:
nope, that makes sense, added, hopefully it is good (I guess not sure how to test that the nightly  configuration is correct locally)